UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentView.cs

```diff
@@ -7,8 +7,34 @@ namespace PlayFab
 
     public class PlayFabMultiplayerAgentView : MonoBehaviour
     {
+        /// <summary>
+        /// Static reference to the current active instance of the PlayFabMultiplayerAgentView.
+        /// </summary>
+        public static PlayFabMultiplayerAgentView current { get; private set; };
+        
         private float _timer;
 
+        /// <summary>
+        /// Awake constructor
+        /// </summary>
+        private void Awake()
+        {
+            Debug.Log($"{Time.fixedTime} PlayFabMultiplayerAgentView awake ");
+            // Check if the static instance already contains a refernce:
+            if (current)
```

Comment:
if(Current != null)  
Certainly the current code works but by being explicit it can help reduces confusion  on whether the variable is a boolean or not.